src/core/freebsd/SDL_evdev_kbd_freebsd.c

```diff
@@ -221,9 +223,12 @@ SDL_EVDEV_keyboard_state *
 SDL_EVDEV_kbd_init(void)
 {
     SDL_EVDEV_keyboard_state *kbd;
+    struct mouse_info mData;
     char flag_state;
     char* devicePath;
 
+    memset(&mData, 0, sizeof(struct mouse_info));
```

Comment:
Use `SDL_zero(mData)` here. It's not life-and-death on FreeBSD, since the C runtime is always available there, but we try to stay in the habit of not using C runtime functions directly on every platform, and clean these up when we find them.